src/main.js

```diff
@@ -507,11 +507,13 @@ class MiniGraphCard extends LitElement {
   }
 
   renderSvg() {
-    const { height } = this.config;
+    const { height, show } = this.config;
+    const grid_lines_type = show.grid_lines_type ? show.grid_lines_type : false;
     return svg`
       <svg width='100%' height=${height !== 0 ? '100%' : 0} viewBox='0 0 500 ${height}'
         @click=${e => e.stopPropagation()}>
         <g>
+          ${grid_lines_type ? this.renderGridLines() : ''}
```

Comment:
Regarding placing "".
In HA Frontend it is customary to use a `nothing` value.
In some my PR to HA frontend I also tried to use "" - but the told me that I must use `nothing`.
Honestly have no idea about a difference.
Do you know anything about it?

Reply:
Huh I've never heard about that, and it's not defined in js.
Setting it to undefined work, tho.

Reply:
OK then, let's leave ""